data/cache/test/test_cache.py

```diff
@@ -28,6 +34,9 @@ def test_caching(cache_type):
 
 
 def test_memcache():
+    global DATA
```

Comment:
What is this madness

Reply:
Allows us to reassign the top level variable; its just for texting

Reply:
Hmm, not a fan but it's okay if just for testing...